src/transactions.rs

```diff
@@ -185,12 +251,18 @@ pub enum Durability {
     /// could cause a transaction to partially commit (some but not all of the data is written).
     /// This is described in the design doc in futher detail.
     ///
+    /// Recovery after a crash is still very slow for large databases, because it needs to read
+    /// the entire database to reconstruct the allocator state.
+    ///
     /// Security considerations: Many hard disk drives and SSDs do not actually guarantee that data
     /// has been persisted to disk after calling `fsync`. Even with this commit level, an attacker
     /// with a high degree of control over the database's workload, including the ability to cause
     /// the database process to crash, can cause the database to crash with the god byte primary bit
     /// pointing to an invalid commit slot, leaving the database in an invalid, potentially attacker-controlled state.
     Paranoid,
+    /// Like [`Durability::Paranoid`], but also saves the allocator state as part of each commit.
+    /// This means repair is no longer required when opening the database after a crash.
+    ParanoidPlus,
```

Comment:
I would suggest `Enclosed` here to convey that this writes more data inline to make the resulting persistent structure self-contained.